packages/react-app-rewired/index.js

```diff
@@ -2,35 +2,47 @@ const path = require('path');
 
 const babelLoaderMatcher = function(rule) {
   return rule.loader && rule.loader.indexOf(`babel-loader${path.sep}`) != -1;
-}
+};
 
 const getLoader = function(rules, matcher) {
   var loader;
 
   rules.some(rule => {
-    return loader = matcher(rule)
+    return (loader = matcher(rule)
       ? rule
-      : getLoader(rule.use || rule.oneOf || [], matcher);
+      : getLoader(rule.use || rule.oneOf || [], matcher));
   });
 
   return loader;
 };
 
 const getBabelLoader = function(rules) {
   return getLoader(rules, babelLoaderMatcher);
-}
+};
 
 const injectBabelPlugin = function(pluginName, config) {
   const loader = getBabelLoader(config.module.rules);
   if (!loader) {
-    console.log("babel-loader not found");
+    console.log('babel-loader not found');
     return config;
   }
-  
+  const plugins = Array.isArray(pluginName) ? pluginName : [pluginName];
```

Comment:
see: https://github.com/timarney/react-app-rewired/pull/121#issuecomment-340188434

Reply:
maybe something like:

```js
const injectBabelPlugin = function(...args) {
  
  const config = args.pop();
  const plugins = args;
  
  const loader = getBabelLoader(config.module.rules);
  if (!loader) {
    console.log('babel-loader not found');
    return config;
  }
  
  // Older versions of webpack have `plugins` on `loader.query` instead of `loader.options`.
  const options = loader.options || loader.query;
  options.plugins = plugins.concat(options.plugins || []);
  
  return config;
}
```

Reply:
It is my fault,  I will fix it